webhook_to_fedora_messaging_messages_messages/__init__.py

```diff
@@ -0,0 +1,132 @@
+
```

Comment:
If these are github-specific, they they should be in a file named `github_utils.py` for example.